File: 코딩 기초 트레이닝/1로 만들기-p181880/1로 만들기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181880

function solution(num_list) {
    var answer = 0;

    num_list.forEach((num) => {
        while(num != 1) {
            if(num % 2 === 0) {
                answer += 1;
                num = num / 2;
            } else {
                num -= 1;
            }
        }
    })
    return answer;
}

File: 코딩 기초 트레이닝/1로 만들기-p181880/1로 만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181880

def solution(num_list):
    answer = 0
    for num in num_list:
        while num != 1:
            if num % 2 == 0 :
                num = num / 2
                answer += 1
            else:
                num -= 1

    return answer
